scrapers/DEU/DE-BY/index.js

```diff
@@ -0,0 +1,79 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as datetime from '../../../lib/datetime.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+// const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  country: 'DEU',
+  state: 'DE-BY', // ISO 3166 notation
+  url: 'https://raw.githubusercontent.com/jgehrcke/covid-19-germany-gae/master/data.csv',
+  type: 'json',
+  timeseries: true,
+  curators: [
+    {
+      name: 'Dr. Jan-Philip Gehrcke',
+      email: 'jgehrcke@googlemail.com'
+    }
+  ],
+  sources: [
+    {
+      name: 'Robert Koch-Institut, Bavarian health ministry',
+      description: 'Fresh data obtained from Bavarian health ministry by ZEIT ONLINE',
+      url: 'https://github.com/jgehrcke/covid-19-germany-gae'
+    }
+  ],
+  async scraper() {
+    const data = await fetch.csv(this.url);
+
+    function rowToResult(row) {
```

Comment:
We've been putting any functions or objects required on the scraper object itself, prefixed with `_`. Just as a matter of consistency, you can move this up to the scraper object:

```
_rowToResult(row) { ...},
```

Then call it with `this._rowToResult()`

Reply:
woop, :rocket: fast feedback, commit coming